department-of-veterans-affairs/caseflow
# Queue | Use CaseDetailsLink in CaseListTable

### Description
This uses the preexisting `<CaseDetailsLink/>` component when navigating case search results, rather than a similar custom implementation.

### Acceptance Criteria 
- [ ] Navigating to appeal detail view from table view or search results works as expected
- [ ] When navigating to detail view from search results, breadcrumbs should be `< Back to ${veteran}'s Case List`; when navigating from table view: `Your Queue > ${veteran}`

### Testing Plan
1. Go to `/queue`, navigate to appeal detail for case. Confirm middle column contains `Assigned by`, `Assigned on`, and `Due` fields
2. Search for an appeal (884668527) from `/queue`, navigate to result. Confirm middle column contains `Assigned to` field
